lib/options/format.js

```diff
@@ -1,7 +1,16 @@
-var systemLineBreak = require('os').EOL;
-
 var override = require('../utils/override');
 
+function getSystemLineBreak() {
+  var systemLinkBreak = '\n';
```

Comment:
Should it be named `systemLineBreak` instead?

Reply:
sure should. I took the quick path and used github's editor to make the changes, and was counting on CI to catch any slips.

Reply:
:+1: let's see if it catches anything